### ISSUE TITLE
unable to navigate to other messages from one card to another 

### ISSUE THREAD

**Issue by prajwalmanju** [microsoft/AdaptiveCards#6781]
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

v1.2

### Problem Description

NVDA: When there are multiple cards for any response unable to navigate to other messages from one card to another using arrow keys
JAWS: Able to move from one message to another with arrow keys up/down
Note: As this accessibity feature is for the people who cannot see there should be a mechanism where user understand multiple messages are there to see. Because User dont know how many messages are there on the scren to view and Tab key is moving to next element but not to next message. Hence user dont know when to use arrow keys.

### Expected Outcome

When there are multiple cards for any response user should able to navigate to other messages from one card to another using arrow keys

### Actual Outcome

When there are multiple cards for any response user unable to navigate to other messages from one card to another using arrow keys

### Card JSON

```JSON
<msg><Adaptive>{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "text": "Here are the top results I found in Knowledge.",
      "wrap": true
    }
  ]
}</Adaptive></msg>{
  VendorRegistrationCarousel
}<msg><Adaptive>{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "text": "You can also perform an advanced search with filters.",
      "wrap": true
    }
  ],
  "actions": [
    {
      "type": "Action.OpenUrl",
      "title": "Perform advanced search",
      "url": "{AdavanceSearchUrl}"
    }
  ]
}</Adaptive></msg>
```

**Comment by Wajeed-msft** [comment]
This issue has been copied over to Microsoft Teams repo here: https://github.com/MicrosoftDocs/msteams-docs/issues/4727
